src/error.rs

```diff
@@ -135,6 +135,51 @@ pub enum ConvertError<A, S, V> {
     Validity(V),
 }
 
+impl<Src, Dst: ?Sized + Unaligned, S, V> From<ConvertError<AlignmentError<Src, Dst>, S, V>>
+    for ConvertError<Infallible, S, V>
```

Comment:
I think we should make this a general `From` conversion of `A`, `S`, and `V`:
```suggestion
impl<Src, Dst: ?Sized + Unaligned, A, S, V, NewA, NewS, NewV>
    From<ConvertError<NewA, NewS, NewV>>
for
    ConvertError<Infallible, S, V>
where
    NewA: From<A>,
    NewS: From<S>,
    NewV: From<V>,
```

Reply:
Hm, this is going to conflict. Let me see if I can make it work...

Reply:
I cannot.